fvtest/porttest/si.cpp

```diff
@@ -2920,6 +2920,120 @@ TEST_F(CgroupTest, sysinfo_get_number_CPUs_cgroup)
 	return;
 }
 #endif /* defined(LINUX) */
+
+/**
+ * Test omrsysinfo_cgroup_subsystem_iterator_* functions.
+ */
+TEST_F(CgroupTest, sysinfo_cgroup_subsystem_iterator)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
+	const char *testName = "omrsysinfo_cgroup_subsystem_iterator";
+
+	reportTestEntry(OMRPORTLIB, testName);
+
+#if defined(LINUX)
```

Comment:
Ref: https://github.com/eclipse/omr/pull/6567#discussion_r897117800
```suggestion
 #if defined(LINUX) && !defined(OMRZTPF)
```